assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -173,11 +173,18 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
     !snapDisabled && (!isDragging || (isDragging && !activeDropTargetId));
   const hideHandles = (isDragging && isMaskable) || Boolean(draggingResource);
 
+  const minWidth = dataToEditorY(resizeRules.minWidth);
```

Comment:
Looks like the minimum sizes aren't currently enforced in the Design Panel, making it still possible to assign values below the minimum, including `0`.

Reply:
I think it's a feature, not a bug 😉  ([see 1.](https://github.com/google/web-stories-wp/pull/993#issuecomment-608133563) + Gutenberg works the same)

Reply:
Oh, alright. Will update the original issue, too, checked that and didn't read through the whole discussion. Thanks for the note!

Reply:
Ah, btw, I've seen some odd behavior when width/height are set to 0 -- perhaps it would still make sense to prevent that? Or at least test that setting to 0 doesn't literally break the editor? For example, if I set the values to 0 and then try to resize or drag it from the available handle-cluster, it gets jumpy and kind of breaks, causing odd values and console errors:
<img width="290" alt="Screenshot 2020-05-20 at 13 41 30" src="https://user-images.githubusercontent.com/3294597/82442327-b972af80-9a9f-11ea-8c2b-866663a44ef5.png">

Perhaps it still shouldn't allow setting to `0` / or alternatively should fix the issues when it's 0? I suspect there might be some dividing with `0` happening somewhere.

cc @pbakaus Thoughts?

Reply:
Created a follow-up issue instead: https://github.com/google/web-stories-wp/issues/1881